src/main/java/org/codehaus/mojo/versions/SetMojo.java

```diff
@@ -204,12 +204,21 @@
 
     /**
      * Whether to update the <code>project.build.outputTimestamp<code> property in the POM when setting version.
+     * Deprecated; please use <code>updateBuildOutputTimestampPolicy</code> instead.
      *
      * @since 2.10
      */
     @Parameter( property = "updateBuildOutputTimestamp", defaultValue = "true" )
     private boolean updateBuildOutputTimestamp;
 
+    /**
+     * Whether to update the <code>project.build.outputTimestamp<code> property in the POM when setting version.
+     *
+     * @since 2.12
+     */
+    @Parameter( property = "updateBuildOutputTimestampPolicy", defaultValue = "onchange" )
+    private String updateBuildOutputTimestampPolicy;
```

Comment:
Please add allowed values to documentation.

Reply:
Done